ts-nextjs-typeorm/external-content.ts

```diff
@@ -0,0 +1,37 @@
+
+
+import * as fs from "fs/promises"
+import * as express from "express";
+
+export const router = express.Router();
+
+
+router.use(express.static("external-content"))
+// router.use('/external-content', express.static("external-content"))
```

Comment:
Is this something that the tutorial will have us uncomment? If not, we should rm it. If yes, I think a comment would be great (to point them to the tutorial and give a brief explanation of what this commented-out bit is for).

Ditto on line 20.